src/modules/postScore.js

```diff
@@ -0,0 +1,26 @@
+import addScore from './addScore.js';
+
+export default async ({ user, score }) => {
```

Comment:
- [ ] I think there is something missing within this function, as right now, I cannot post any score to the API. I just took a look and saw that I can see your scores, but whenever I create a new one, there is no response. Here is an image of the app and the API scores that are currently online:
![Captura de pantalla 2022-05-27 070549](https://user-images.githubusercontent.com/59100477/170633149-239ffd1a-c516-4dd1-afa4-5b3f5d977f0b.jpg)

![Captura de pantalla 2022-05-27 070646](https://user-images.githubusercontent.com/59100477/170633151-f58a880a-f0ce-4fb7-8061-c4fb1413ab41.jpg)

I tried posting scores several times but none of them worked, so please work on this issue.

Reply:
please its adding to the API that's why you can see it. kindly check your connections please or your contact me on slack thanks